iterators.tex

```diff
@@ -623,31 +650,186 @@
 \textit{p}).
 \exitexample
 
-\indexlibrary{\idxcode{Iterator}}%
-\begin{codeblock}
+{\color{newclr}
+\rSec2[iterators.sizediterator]{Concept SizedIterator}
+\pnum
+The \tcode{SizedIterator} concept refines \tcode{Iterator} with the
+capability to determine the distance between two iterators that denote
+elements of the same range in constant time with the \tcode{-} operator.
+
+\indexlibrary{\idxcode{SizedIterator}}%
+\begin{itemdecl}
+  template <class>
+  constexpr bool disable_sized = false;
```

Comment:
Regarding `SizedIterator`, the committee has expressed reservations at the sheer number of concepts. I'd rather have fewer than more. I see `SizedIterator` as one we can do without.

> What's the use case for `disable_sized_sentinel`?

I'm concerned about an unconstrained and unrelated `op-` getting found by ADL. ADL has trained me to be paranoid. Besides, there is the symmetry argument. There are two types involved in the expression `s-i`. The `disable` trait should take two parameters.